src/utils/index.js

```diff
@@ -244,7 +244,10 @@ export function acceptPropAsAcceptAttr(accept) {
   if (isDefined(accept)) {
     return (
       Object.entries(accept)
-        .reduce((a, [mimeType, ext]) => [...a, mimeType, ...ext], [])
+        .reduce((a, [mimeType, ext]) => {
+          const validExt = ext.filter(isExt);
+          return [...a, ...(validExt.length > 0 ? validExt : [mimeType])];
+        }, [])
```

Comment:
I'm not quite sure what this change is supposed to achieve. The test you changed will pass without this change.